README.md

```diff
@@ -52,6 +52,9 @@ export default {
 }
 ```
 
+### Typescript
+Add `@nuxtjs/moment` to the `types` array in your `tsconfig.json`.
```

Comment:
Show how and why, see https://axios.nuxtjs.org/setup#typescript-setup

Reply:
I added a description very similar to the one axios provides in 4f3acb76f1cac6422e2181be3ec728b4ff35641d

Reply:
It would be nice to inform the user that from nuxt 2.9 it is `@nuxt/types` and not `@nuxt/vue-app`

Reply:
Similar to warning https://github.com/nuxt-community/moment-module#setup

Reply:
Missed that one, thanks for the hint!

Reply:
I changed the setup to be valid for nuxt > 2.9 and added a warning for older nuxt versions.